internal/project/service.go

```diff
@@ -305,11 +305,11 @@ func (s *Service) onConfigFileChanged(project *Project, changeKind lsproto.FileC
 		project.deferredClose = true
 	}
 
-	s.delayUpdateProjectGraph(project)
 	if !project.deferredClose {
 		project.pendingReload = PendingReloadFull
 		project.markAsDirty()
 	}
+	project.updateIfDirty()
```

Comment:
Seems to me this shouldnt be called here and should either send notification to client that things have changed and/or update on next request ?

Reply:
I think we’ll need this because we’ll have to do push diagnostics for config file errors, which aren’t hooked up yet, but I think we could do that for now. I think I’d like to do that in a separate PR though, as this one is just trying to prove that there was a bug and fix it. We’re trying to get things stable for an npm release.